src/org/sosy_lab/java_smt/example/AllSatExample.java

```diff
@@ -1,3 +1,11 @@
+// This file is part of JavaSMT,
+// JavaSMT is an API wrapper for a collection of SMT solvers.:
+// https://github.com/sosy-lab/java-smt
+//
+// SPDX-FileCopyrightText: 2020 Dirk Beyer <https://www.sosy-lab.org>
+//
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Are these and other examples intended to be just for looking at them, or are they intended as template code that can be copied into own projects and changed/extended there?

If the latter is desired, we might want to use a license with less restrictions.

Reply:
Of course this code can be copied and modified. Which license is appropriate here? CC?

Reply:
What license do you mean by CC? This is not an SPDX identifier, nor a common abbreviation.

Of course the copyright holder needs to decide this.

The minimal amount of restrictions (i.e., none) are contained in CC-0 and Unlicense. Both are widely used, although the OSI [recommends not using them](https://opensource.org/faq#public-domain).

One solution would be to use something like `Unlicense OR MIT`. This way, users can pick whether they want no restrictions or prefer an OSI-approved license. Of course we could make the disjunction even broader if we want to.